app/services/watsonSpeech.js

```diff
@@ -0,0 +1,39 @@
+const TextToSpeechV1 = require("ibm-watson/text-to-speech/v1");
+const {IamAuthenticator} = require("ibm-watson/auth");
+const fs = require("fs");
+
+module.exports = function() {
```

Comment:
Use ES6 callback function declaration: `() => {`

Reply:
We use `.eslintrc.json` for rules concerning formatting. Please make according changes to this file if you want to enforce specific code styles, refactor the existing files to match this style and create a new branch.